model/src/test/kotlin/ProjectTest.kt

```diff
@@ -27,6 +27,11 @@ import io.kotlintest.specs.WordSpec
 import java.io.File
 import java.time.Instant
 
+private fun readAnalyzerResults(analyzerResultsFilename: String): Project =
```

Comment:
While at it, we should probably rename all occurrences of "AnalyzerResult*s*" (plural) with "AnalyzerResult" (singular), i.e. `readAnalyzerResults` -> `readAnalyzerResult` and `analyzerResultsFilename` -> `analyzerResultFilename`, to match the `ProjectAnalyzerResult` class name.

Reply:
Done.